From c097349b42099784319f3f7a3836ad7de74a0d83 Mon Sep 17 00:00:00 2001
From: Dmitry Bogatov <KAction@gnu.org>
Date: Thu, 22 Mar 2018 23:06:49 +0300
Subject: [PATCH] Make bin:dput-ng recommend py3 version of paramiko

Since dput-ng is written in Python3, there is no use from extra Python2
library. python3-paramiko is needed for `scp` method, which, although
considered deprecated by dput-ng, is recommended by
debomatic-*.debian.net services.
---
 debian/control | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control b/debian/control
index 55d3455..bf38540 100644
--- a/debian/control
+++ b/debian/control
@@ -43,7 +43,7 @@ Depends:
  ${python3:Depends},
 Recommends:
  bash-completion,
- python-paramiko,
+ python3-paramiko,
 Description: next generation Debian package upload tool
  dput-ng is a Debian package upload tool which provides an easy to use
  interface to Debian (like) package archive hosting facilities. It allows
